--- rustc-1.11.0/src/etc/get-stage0.py.no-download	2016-08-24 00:34:03.561598809 -0700
+++ rustc-1.11.0/src/etc/get-stage0.py	2016-08-24 00:34:08.636555684 -0700
@@ -31,9 +31,8 @@ def main(triple):
     filename = 'rustc-{}-{}.tar.gz'.format(channel, triple)
     url = 'https://static.rust-lang.org/dist/{}/{}'.format(date, filename)
     dst = dl_dir + '/' + filename
-    if os.path.exists(dst):
-        os.unlink(dst)
-    bootstrap.get(url, dst)
+    if not os.path.exists(dst):
+        bootstrap.get(url, dst)
 
     stage0_dst = triple + '/stage0'
     if os.path.exists(stage0_dst):
